sqlglot/parser.py

```diff
@@ -377,6 +377,8 @@ class Parser(metaclass=_Parser):
         TokenType.CACHE: lambda self: self._parse_cache(),
         TokenType.UNCACHE: lambda self: self._parse_uncache(),
         TokenType.USE: lambda self: self._parse_use(),
+        TokenType.START: lambda self: self._parse_transaction(),
```

Comment:
there shouldn't be a token start, i think dialects should just map start to begin